### ISSUE TITLE
VENTCRAWLING IS FUCKING BROKEN

### ISSUE THREAD

**Issue by Buggy123** [tgstation/tgstation#33717]
STEP 1: BE IN PIPE
STEP 2: MOVE
STEP 3: RANDOMLY POP OUT OF THE VENT AT THE WORST POSSIBLE TIMES.

ROUND ID: ANY FUCKING ROUND ITS NOT ROUND DEPENDENT.

**Comment by GunHog** [comment]
I tested this as a human, with my "ventcrawler" variable set to 2. I was ejected a total of three times from the distro lines, two times which occurred in a corner pipe. I appeared above the pipe, with no vent nearby. The third time is the strangest, where I was _transferred from the distro pipes to the waste pipes._ 

I have not yet to discern a pattern connecting these three events, besides corner pipes in the first two, but I was in a straight pipe when I was transferred from the distro pipe to the scrubbers pipe. These pipes were on the same tile.
